### PR TITLE
Feature/issue #14 encrypt

### PR DESCRIPTION
**Pull request by enfycius**

### CODE REVIEW

**Review by devdubu — APPROVED** [review]
오키

File: views/list.ejs [hunk 1]
```diff
@@ -64,13 +74,51 @@
...
+            method: 'POST',
+            body: new URLSearchParams(new FormData(event.target)) // event.target is the form
+        }).then((resp) => {
+            return resp.json(); // or resp.text() or whatever the server sends
+        }).then((body) => {
+            // TODO handle body
+            console.log(body);
+        }).catch((error) => {
+            // TODO handle error
+        });
```
Comment (enfycius) [inline views/list.ejs:97]: it is unnecessary.

File: views/list.ejs [hunk 2]
```diff
@@ -64,13 +74,51 @@
...
+
+        window.onclick = function(event) {
+            var modal = document.getElementById("Modal");
+            if (event.target == modal) {
+                modal.style.display = "none";
+            }
+        }
+});
```
Comment (enfycius) [inline views/list.ejs:105]: It is also.

Comment (enfycius) [inline views/list.ejs:105]:
```javascript
});
```
It is also.

**Pull request merged**